src/components/production-line/user-list.tsx

```diff
@@ -138,13 +141,17 @@ export const UserList = ({
                     audioLevelAboveThreshold && p.endpointId === dominantSpeaker
                   }
                 >
-                  <OnlineIndicator isActive={p.isActive}>
-                    <UserIcon />
+                  <OnlineIndicator
+                    className={
+                      p.isActive ? (p.isWhip ? "whip" : "active") : "inactive"
+                    }
```

Comment:
Maybe update the classname for users that are not WHIP now it's a bit weird, since a WHIP-stream can be active but is not included in this classname.